queries/orderbook/batch_rewards.sql

```diff
@@ -70,7 +70,7 @@ WITH observed_settlements AS (SELECT
                                    surplus + fee - reference_score as uncapped_reward_eth,
                                    -- Uncapped Reward = CLAMP_[-E, E + exec_cost](uncapped_reward_eth)
```

Comment:
Is there a typo here in the comment? You write
`Uncapped Reward = CLAMP ... (uncapped_reward_eth`

Reply:
Ya definitely. Left side should be capped reward. Gonna have to fix this later.